stko/calculators/torsion_calculators.py

```diff
@@ -135,3 +137,74 @@ def get_results(self, mol):
             generator=self.calculate(mol),
             mol=mol,
         )
+
+
+class MatchedTorsionCalculator(ConstructedMoleculeTorsionCalculator):
+    """Matches rdkit generated torsions with building block torsions.
+    """
+
+    def calculate(self, mol: stk.ConstructedMolecule):
+        """extract torsions with rdkit, then match to building blocks
```

Comment:
Doc strings and comments should be full sentences: `Extract torsions with rdkit, then match to building blocks.`